pkg/cmd/printer_test.go

```diff
@@ -183,3 +183,121 @@ Bob-and-Eve-can-view-pods  ClusterRole/view  Eve      User
 	}
 
 }
+
+func TestPrinter_ExportData(t *testing.T) {
+	testCases := []struct {
+		scenario string
+
+		verb           string
+		resource       string
+		nonResourceURL string
+		resourceName   string
+
+		roleBindings        []rbac.RoleBinding
+		clusterRoleBindings []rbac.ClusterRoleBinding
+
+		wide   bool
+		output string
+	}{
+		{
+			scenario: "A",
+			verb:     "get", resource: "pods", resourceName: "",
+			output: "{}\n",
+		},
+		{
+			scenario: "B",
+			verb:     "get", resource: "pods", resourceName: "my-pod",
+			output: "{}\n",
+		},
+		{
+			scenario: "C",
+			verb:     "get", nonResourceURL: "/healthz",
+			output: "{}\n",
+		},
+		{
+			scenario: "D",
+			verb:     "get", resource: "pods",
+			roleBindings: []rbac.RoleBinding{
+				{
+					ObjectMeta: meta.ObjectMeta{Name: "Alice-can-view-pods", Namespace: "default"},
+					Subjects: []rbac.Subject{
+						{Name: "Alice", Kind: "User"},
+					}},
+				{
+					ObjectMeta: meta.ObjectMeta{Name: "Admins-can-view-pods", Namespace: "bar"},
+					Subjects: []rbac.Subject{
+						{Name: "Admins", Kind: "Group"},
+					}},
+			},
+			clusterRoleBindings: []rbac.ClusterRoleBinding{
```

Comment:
Related to my previous comment, it would be nice to test that we do not print ClusterRoles/ClusterRoleBinding that are not associated with any Subject.

Reply:
Done!